### PR TITLE
Refactored client event if-else cascade to switch statement

### PR DESCRIPTION
**Pull request by bendeutsch**
This change had been requested in #4524, but it is out of scope for such a long-lived feature branch. I'm hereby moving it to its own pull request (i.e. this one).

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -3346,10 +3377,20 @@ void Game::processClientEvents(CameraOrientation *cam)
...
 			delete event.set_sky.params;
-		} else if (event.type == CE_OVERRIDE_DAY_NIGHT_RATIO) {
-			bool enable = event.override_day_night_ratio.do_override;
-			u32 value = event.override_day_night_ratio.ratio_f * 1000;
-			client->getEnv().setDayNightRatioOverride(enable, value);
+			break;
+
+		case CE_OVERRIDE_DAY_NIGHT_RATIO:
+			{
+				bool enable = event.override_day_night_ratio.do_override;
```
Comment (nerzhul) [inline src/game.cpp]: not needed temporary variable, just use the variable in function

File: src/game.cpp [hunk 2]
```diff
@@ -3159,21 +3162,28 @@ void Game::processClientEvents(CameraOrientation *cam)
 			player->hurt_tilt_strength =
 				rangelim(event.player_damage.amount / 4, 1.0, 4.0);
 
-			MtEvent *e = new SimpleTriggerEvent("PlayerDamage");
-			client->event()->put(e);
-		} else if (event.type == CE_PLAYER_FORCE_MOVE) {
+			{
+				MtEvent *e = new SimpleTriggerEvent("PlayerDamage");
```
Comment (nerzhul) [inline src/game.cpp]: unneeded temp variable

File: src/game.cpp [hunk 3]
```diff
@@ -3245,74 +3266,84 @@ void Game::processClientEvents(CameraOrientation *cam)
...
-				delete event.hudchange.v2s32data;
-				continue;
+			break;
+
+		case CE_HUDRM:
+			{
+				HudElement *e = player->removeHud(event.hudrm.id);
+
+				if (e != NULL)
+					delete(e);
```
Comment (nerzhul) [inline src/game.cpp]: please remove those parenthesis

**Comment by bendeutsch** [comment]
Fixed and pushed with `commit --amend`.

**Comment by nerzhul** [comment]
@bendeutsch just do a second commit we will squash ourselved

**Comment by bendeutsch** [comment]
Try it now, please.

**Comment by nerzhul** [comment]
will merge this this evening if not already done

**Comment by paramat** [comment]
e7246d3f0deed22f7d81e8988f656c594720d672

**Pull request closed without merging**